### PR TITLE
Update inifile Consolidation instructions

### PR DESCRIPTION
**Pull request by peterchoo**
## Problem

Application INI File consolidation instructions require updating to add in a couple more instructions regarding product IDs. 

## Solution

This PR Updates the inifile consolidation instructions adding in instructions for creating the header for the file, and adjusting the workflow to reduce user repetition.

In addition, an examples folder for S/4HANA 1809 SP5 has been created, containing a completed consolidated inifile.

## Tests

- [x] Doc changes LGTM

### CODE REVIEW

File: documentation/ansible/system-design-deployment/app/prepare-ini.md [hunk 1]
```diff
@@ -372,63 +372,125 @@ _**Note:** Steps prefixed with * may not be encountered in 2020 versions of SAP
...
-   1. Copy in all uncommented values from the SCS template and group key value pairs based on prefix, e.g. `NW_SCS_Instance`
-   1. Copy in all uncommented values from the DB template and group key value pairs based on prefix, e.g. `NW_HDB_DB`
-   1. Copy in all uncommented values from the PAS template and group key value pairs based on prefix, e.g. `NW_CI_Instance`
-   1. Copy in all uncommented values from the AAS template and group key value pairs based on prefix, e.g. `NW_DI_Instance`
-   1. Deduplicate any common key value pairs.
+To use the inifiles during the installation process, they should be consolidated into one file.
+
+The overall process is to download the files to your workstation, extract the uncommented key value pairs into a new file, organise for ease of reference later, update values to Ansible variables for use with automation.
+
+The file should be saved with meaningful name relating to the SAP Product e.g `S4HANA_2020_ISS_v001.inifile.params` and uploaded to the Storage Account.
```
Comment (robfallows) [inline documentation/ansible/system-design-deployment/app/prepare-ini.md:379]:
```suggestion
The file should be saved with a meaningful name relating to the SAP Product e.g `S4HANA_2020_ISS_v001.inifile.params` and uploaded to the Storage Account.
```

File: documentation/ansible/system-design-deployment/app/prepare-ini.md [hunk 2]
```diff
@@ -372,63 +372,125 @@ _**Note:** Steps prefixed with * may not be encountered in 2020 versions of SAP
...
+
+1. Download all `inifile.params` files to your workstation, and make a backup of each file.
+1. In **your editor** open each file.
+1. Create a new consolidation file named for the SAP Product, e.g. `S4HANA_2020_ISS_v001.inifile.params`.
+1. From the SCS inifile, copy the header into the consolidated and format for readability, e.g.
```
Comment (robfallows) [inline documentation/ansible/system-design-deployment/app/prepare-ini.md:384]:
```suggestion
1. From the SCS inifile, copy the header into the consolidated file and format for readability, e.g.
```

File: documentation/ansible/system-design-deployment/app/prepare-ini.md [hunk 3]
```diff
@@ -372,63 +372,125 @@ _**Note:** Steps prefixed with * may not be encountered in 2020 versions of SAP
...
     1. In the Azure Portal navigate to the `sapbits` container
-    1. Create a new `templates` directory under `sapbits`
-    1. click "Upload"
+    1. Click "Upload"
     1. In the panel on the right, click Select a file
-    1. Navigate your workstation to the template generation directory `/tmp/app_template`
     1. Select the generated template, e.g. `S4HANA_2020_ISS_v001.inifile.params`
-    1. click "Advanced" to show the advanced options, and enter `templates` for the Upload Directory
-    1. click "Upload"
+    1. Click "Upload"
```
Comment (robfallows) [inline documentation/ansible/system-design-deployment/app/prepare-ini.md:482]: Needs to upload to the `templates/` folder.

**Review by robfallows — CHANGES_REQUESTED** [review]
Some small changes needed

File: documentation/ansible/system-design-deployment/app/prepare-ini.md (hunk 3 above)
Comment (peterchoo) [inline documentation/ansible/system-design-deployment/app/prepare-ini.md:482]:
```suggestion
    1. Click "Upload"
    1. Click "Advanced" to show the advanced options, and enter `templates` for the Upload Directory
```

**Review by robfallows — APPROVED** [review]
LGTM 👍

**Pull request merged**